batch/batch/api.py

```diff
@@ -74,76 +93,13 @@ def create_batch(self, url, attributes, callback, ttl):
             doc['callback'] = callback
         if ttl:
             doc['ttl'] = ttl
-        response = requests.post(url + '/batches/create', json=doc, timeout=self.timeout)
-        raise_on_failure(response)
-        return response.json()
+        return self.post(url + '/batches/create', json=doc)
 
     def get_batch(self, url, batch_id):
-        response = requests.get(url + '/batches/{}'.format(batch_id), timeout=self.timeout)
-        raise_on_failure(response)
-        return response.json()
+        return self.get(url + f'/batches/{batch_id}')
 
     def close_batch(self, url, batch_id):
-        response = requests.post(url + '/batches/{}/close'.format(batch_id), timeout=self.timeout)
-        raise_on_failure(response)
-        return response.json()
+        return self.post(url + f'/batches/{batch_id}/close')
 
     def delete_batch(self, url, batch_id):
-        response = requests.delete(url + '/batches/{}'.format(batch_id), timeout=self.timeout)
-        raise_on_failure(response)
-        return response.json()
-
-    def refresh_k8s_state(self, url):
```

Comment:
Want to check that this Api method was meant to go

Reply:
If so, https://github.com/hail-is/hail/blob/80d7c9020b99fc446640abc2c2f2068669347d71/batch/batch/client.py#L209 and the corresponding client test need to be removed.